docs/integrations/destinations/bigquery.md

```diff
@@ -111,15 +111,13 @@ This is the recommended configuration for uploading data to BigQuery. It works b
 * **GCS Bucket Path**
 * **Block Size (MB) for GCS multipart upload**
 * **GCS Bucket Keep files after migration**
-  * See [this](https://cloud.google.com/storage/docs/creating-buckets) for instructions on how to create a GCS bucket.
+  * See [this](https://cloud.google.com/storage/docs/creating-buckets) for instructions on how to create a GCS bucket. The bucket cannot have a retention policy. Set Protection Tools to none or Object versioning.
 * **HMAC Key Access ID**
-  * See [this](https://cloud.google.com/storage/docs/authentication/hmackeys) on how to generate an access key.
-  * We recommend creating an Airbyte-specific user or service account. This user or account will require read and write permissions to objects in the bucket.
+  * See [this](https://cloud.google.com/storage/docs/authentication/managing-hmackeys) on how to generate an access key. For more information on hmac keys please reference the [GCP docs](https://cloud.google.com/storage/docs/authentication/hmackeys)
+  * We recommend creating an Airbyte-specific user or service account. This user or account will require the following permissions for the bucket: `Storage Object Admin` and `Storage Admin`. You can set those by going to the permissions tab in the GCS bucket and adding the appropriate the email address of the service account or user and adding the aforementioned permissions.
```

Comment:
@tuliren I updated this line after you reviewed it. In practice I found `Storage Object Admin` wasn't enough permission and needed to add `Storage Admin`. Relates to this bug [issue](https://github.com/airbytehq/airbyte/issues/9044). Are you okay with me publishing this as is or is there a more limited permission that you know works?

Reply:
Hi ✋ what was the error you saw after adding the `storage.multipartUploads.create` permission but it still didnt work? I'm going to guess it was something about not seeing a bucket, and the actual permission you needed was `storage.buckets.list`. Storage Admin probably wouldnt be the best thing to suggest people give to this SA.

I've always felt that the default BigQuery/Storage Roles GCP creates for users dont include some basic permissions I think they should. And then people end up granting an Admin role because of this. I usually create a custom role "Storage Viewer" with this:

<img width="268" alt="Screen Shot 2022-01-03 at 2 28 53 PM" src="https://user-images.githubusercontent.com/68556134/147987325-1fba55cc-9de6-4642-8186-d89883ed6859.png">

And use it in combination with one of the Storage Object _ roles

Reply:
> Are you okay with me publishing this as is or is there a more limited permission that you know works?

I am fine with merging this PR as is. I don't know a more limited permission that works.